test/e2e-tests/README.md

```diff
@@ -1,16 +1,33 @@
 # AI Edge testing
 
-Place for testing and to verify the MLOps pipelines are working correctly. These tests use the `k8s.io/client-go` package to interact with the cluster. Using `oc login` to log into the cluster as normal should mean no setup is needed to run the tests.
+Place for testing and to verify the MLOps pipelines are working correctly. These tests use the `k8s.io/client-go` package to interact with the cluster. Using `oc login` to log into the cluster as normal should mean little setup is needed to run the tests.
 
 A local install of the Go compiler is needed to run the tests. Go version `1.21` is required to run the tests.
 
+## Setup
+- Log into the target cluster using `oc login`. This will update your default `kubeconfig` for the tests to use
+- Create a S3 bucket with the models in the root directory
+- Create a quay.io robot
```

Comment:
```suggestion
```

This one isn't a hard requirement. While it will be what is usually done, someone could pass a user's username/password, or use a different registry that doesn't have a concept of robot accounts.